bucket/mro.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": "3.5.1",
+    "description": "Microsoft R Open: The enhanced distribution of R from Microsoft Corporation",
+    "homepage": "https://mran.microsoft.com/",
+    "license": "GPL-2.0-only",
+    "depends": [
+        "wixtoolset",
+        "extras/vcredist2015"
```

Comment:
Remove vcredist from depends and move it to suggest.

Reply:
VC Runtime 2015 is a must-install component during MRO installation (the checkbox is gray), so vcredist2015 is put to `depends` instead of `suggest`.
![mro](https://user-images.githubusercontent.com/5832170/50763040-aa73ce80-12a9-11e9-9c81-de9f36822e11.jpg)

Reply:
vcredist2015 is under `suggest` now.